tests/lib/rules/context-display-name.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @fileoverview Prevent missing displayName in a React context definition
+ * @author Jules Blom
+ */
+
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../../../lib/rules/context-display-name');
+
+const parsers = require('../../helpers/parsers');
+
+const parserOptions = {
+  ecmaVersion: 2018,
+  sourceType: 'module',
+  ecmaFeatures: {
+    jsx: true,
+  },
+};
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({ parserOptions });
+ruleTester.run('context-display-name', rule, {
+  valid: parsers.all([
+    {
+      code: `
+        import React from 'react';
+
+        const Hello = React.createContext();
+        Hello.displayName = "HelloContext"
+      `,
+    },
+    {
+      code: `
+        import { createContext } from 'react';
+
+        const Hello = createContext();
+        Hello.displayName = "HelloContext"
+      `,
+    },
+    {
+      code: `
+        import { createContext } from 'react';
+
+        const Hello = createContext();
+
+        const obj = {};
+        obj.displayName = "False positive";
+
+        Hello.displayName = "HelloContext"
+      `,
+    },
+    {
+      code: `
+        import * as React from 'react';
+
+        const Hello = React.createContext();
+
+        const obj = {};
+        obj.displayName = "False positive";
```

Comment:
should there be some of these test cases that avoid React.createContext entirely?

Reply:
Just like this?

```javascript
const obj = {};
obj.displayName = "False positive";
```

Reply:
yup, looks good!

Reply:
OK added!